src/Http/Http.Extensions/src/SendFileResponseExtensions.cs

```diff
@@ -117,7 +117,7 @@ private static async Task SendFileAsyncCore(HttpResponse response, IFileInfo fil
                     {
                         fileContent.Seek(offset, SeekOrigin.Begin);
                     }
-                    await StreamCopyOperation.CopyToAsync(fileContent, response.Body, count, cancellationToken);
+                    await StreamCopyOperation.CopyToAsync(fileContent, response.BodyWriter, count, cancellationToken);
```

Comment:
My guess is this will be good for kestrel but bad for IIS/HTTP.SYS.

Reply:
Http.Sys implements SendFile so this code path is never hit there. I can't recall if IIS implements the feature.

Reply:
I forgot all servers now implement SendFile so the only change would be if somebody call the extension method explicitly called with an IFileInfo that doesn't have a physical path 😄